README.md

```diff
@@ -35,6 +35,25 @@ All API methods that return lists (`getMarkets`, `getOrders`, `getOrderFills`, a
 
 All of these methods also take a optional `page` argument, which you can use to fetch additional pages.
 
+## Shares primer
+
+Veil markets each have two tokens: LONG and SHORT. By holding shares of LONG or SHORT tokens, you hold a "position" in a market. When a market ends, its LONG and SHORT shares are redeemable for ETH, with the rates depending on the market's result. 
+
+In yes/no markets (such as, "Will ETH be above $100 at the end of 2018?"), the payout goes entirely to one share token, LONG if the market resolves to "Yes", and SHORT if the market resolves to "No".
```

Comment:
`In yes/no markets (e.g. "Will ETH be above $100 at the end of 2018?"), the payout goes entirely to one share token—LONG if the market resolves to "Yes" and SHORT if the market resolves to "No".`